docs/articles/expensify-classic/billing-and-subscriptions/Pay-Per-Use-Subscription.md

```diff
@@ -1,5 +1,29 @@
 ---
 title: Pay-per-use Subscription
-description: Pay-per-use Subscription
+description: Learn more about your pay-per-use subscription. 
 ---
-## Resource Coming Soon!
+# Overview
+Pay-per-use is a billing option for people who prefer to use Expensify month to month or on an as-needed basis. On a pay-per-use subscription, you will only pay for active users in that given month. 
+
+**We recommend this billing setup for companies that use Expensify a few months out of the year**. If you have expenses to manage for more than 6 out of 12 months, an [**Annual Subscription**](https://help.expensify.com/articles/expensify-classic/billing-and-subscriptions/Annual-Subscription#gsc.tab=0)  may better suit your needs. 
+
+# How to start a pay-per-use subscription 
+1. Create a Group Workspace if you haven’t already by going to **Settings > Workspaces > Group > New Workspace** 
+2. Once you’ve created your Workspace, under the “Subscription” section on the Group Workspace page, select “Pay-per-use”.
+
+# FAQ
+
+## What is considered an active user? 
+An active user is anyone who chats, creates, modifies, submits, approves, reimburses, or exports a report in Expensify. This includes actions taken by a Copilot and Workspace automation (such as Scheduled Submit and automated reimbursement). If no one on your Group Workspace uses Expensify in a given month, you will not be billed for that month.
```

Comment:
> An active user is anyone who chats

@shmaxey Isn't chat free for users and just managing expenses is paid?

Reply:
For a corporate/control plan, no - I believe they get three interactions for free. Free workspaces get chat for free (but this is a pay-per-use article, so only control/coporate plans here!)